ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -747,6 +799,159 @@ export class ObjectDetailComponent implements OnInit {
     this.addedPkColumnName = columnName
   }
 
+  addCcColumn() {
+    const index = this.ccData.length;
+  
+    this.ccData.push({
+      spSno: (index + 1).toString(),
+      spConstraintName: `Constraint_name${index + 1}`,
+      spCondition: '',
+      srcSno: '',
+      srcCondition: '',
+      srcConstraintName: '',
+      deleteIndex: `cc${index + 1}`,
```

Comment:
What is delete index?

Reply:
Is it like the id associated in the backend?

Reply:
this Id been used to delete the check constraint when click on remove constraint because it will remain change inrespective of order of check constraint